diff --git a/libtiff/libtiff_ctypes.py b/libtiff/libtiff_ctypes.py
index 62eb271..4d9c118 100644
--- a/libtiff/libtiff_ctypes.py
+++ b/libtiff/libtiff_ctypes.py
@@ -114,6 +114,8 @@
         if len(words) != 2:
             continue
         name, value = words
+        if name in ['TIFF_GCC_DEPRECATED', 'TIFF_MSC_DEPRECATED']:
+            continue
         i = value.find('/*')
         if i != -1:
             value = value[:i]

